mllib/src/main/scala/org/apache/spark/ml/stat/Summarizer.scala

```diff
@@ -397,17 +398,12 @@ private[spark] object SummaryBuilderImpl extends Logging {
 
     override def serialize(state: SummarizerBuffer): Array[Byte] = {
       // TODO: Use ByteBuffer to optimize
```

Comment:
If there is no clear optimization goal, can we remove this TODO now? @zhengruifeng

Reply:
LGTM

actually this TODO originated from https://github.com/apache/spark/commit/07549b20a3fc2a282e080f76a2be075e4dd5ebc7 , so cc @WeichenXu123